modules/ingest-geoip/qa/full-cluster-restart/build.gradle

```diff
@@ -37,7 +37,9 @@ tasks.withType(Test).configureEach {
   }
 }
 
-BuildParams.bwcVersions.withWireCompatible(v -> v.before("8.0.0")) { bwcVersion, baseName ->
+// once we are ready to test migrations from 8.x to 9.x, we can set the compatible version to 8.0.0
+// see https://github.com/elastic/elasticsearch/pull/93666
+BuildParams.bwcVersions.withWireCompatible(v -> v.before("7.0.0")) { bwcVersion, baseName ->
```

Comment:
Should we add in an assert here for v.before(v8+1 major)? Not sure it's possible at all with Gradle scripts.

Reply:
I added an assertion -- hope it satisfies!